todo_app/data/trello_board.py

```diff
@@ -0,0 +1,79 @@
+import os, requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+trello_key = os.getenv('TRELLO_KEY')
+trello_token = os.getenv('TRELLO_TOKEN')
+
+class TrelloBoard:
+
+    cards = []
+    lists = []
+
+    def __init__(self, board_id):
+        self.board_id = board_id
+
+    def parse_get_cards_response(self,get_cards_response):
+        self.cards = []
+        for card in get_cards_response:
+                self.cards += [Card(card['id'],card['idList'],card['name'])]
+        return self.cards
+
+
+    def get_cards(self):
+        cards_url = 'https://api.trello.com/1/boards/' + self.board_id + '/cards'
+        print(cards_url)
+        get_cards_response = requests.get(cards_url, params={ 'key':trello_key,  'token' : trello_token})
+        
+        self.parse_get_cards_response(get_cards_response.json())
+
+        return self.cards
+
+
+    def parse_get_lists_response(self,get_lists_response):
+        self.lists = []
+        for list in get_lists_response:
+            self.lists += [List(list['id'],list['name'])]
+        return self.lists
+
+
+    def get_lists(self):
+        lists_url = 'https://api.trello.com/1/boards/' + self.board_id + '/lists'
+        print(lists_url)
+        get_lists_response = requests.get(lists_url, params={ 'key':trello_key,  'token' : trello_token})
```

Comment:
Getting the key and token in the params is a little repetitive. You could instead have a self.auth_params property or similar that gives you that dictionary. Your various functions can get a copy and add extra params if necessary.